File: 프로그래머스 level 1/최대공약수와 최소공배수.py
def solution(n, m):
    table = []
    for i in range(1, min(n, m) + 1):
        if n%i==0 and m%i==0:
            table.append(i)
    a = max(table)

    if min(n, m) == a:
        b = max(n, m)
    else:
        b = a * (n / a) * (m / a)
    return [a,b]

File: Swift/프로그래머스 level 1/최대공약수와 최소공배수.swift
func solution(_ n:Int, _ m:Int) -> [Int] {
    var answer: [Int] = []
    var gcd = min(n, m)
    
    while gcd > 0 {
        if n % gcd == 0 && m % gcd == 0 {
            break
        } else {
            gcd -= 1
        }
    }
    
    return [gcd, gcd * (n / gcd) * (m / gcd)]
}